packages/react-cosmos/src/server/web/export.js

```diff
@@ -87,8 +92,17 @@ export async function generateExport() {
         console.log(outputPath);
       },
       err => {
-        console.error('[Cosmos] Export Failed! See error below:');
-        console.error(err);
+        console.error('[Cosmos] Export Failed! See errors below:');
+        if (err.webpackErrors) {
+          err.webpackErrors.forEach(error => {
+            console.error('');
+            console.error(error);
+          });
+          console.error('');
```

Comment:
Can we remove this? I don't think it's needed and isn't consistent with the `else` branch anyway.